### PR TITLE
[api] don't crash when a project meta can't get imported from backend

### PR DESCRIPTION
**Pull request by adrianschroeter**

### CODE REVIEW

File: src/api/app/jobs/consistency_check.rb [hunk 1]
```diff
@@ -55,7 +55,11 @@ def check_project(fix = nil)
       @errors << project_meta_check(project, fix)
     rescue Project::UnknownObjectError
       # specified but does not exist in api. does it also not exist in backend?
-      @errors << import_project_from_backend(ENV['project'])
+      answer = import_project_from_backend(ENV['project'])
+      unless string.blank?
```
Comment (DavidKang) [inline src/api/app/jobs/consistency_check.rb]: This should be `answer.blank`? :thinking:

Comment (adrianschroeter) [inline src/api/app/jobs/consistency_check.rb]: right, fixed...

**Comment by hennevogel** [comment]
@DavidKang can you please re-review?

**Comment by hennevogel** [comment]
@DavidKang hey?

**Comment by DavidKang** [comment]
@hennevogel sure!

**Comment by DavidKang** [comment]
@adrianschroeter, when you can, please rebase to solve the conflicts :smile_cat:.

**Comment by hennevogel** [comment]
@adrianschroeter please rebase

**Comment by bgeuken** [comment]
Rebased in #3611

**Pull request closed without merging**